portworx-manual/temp-px-deploy/kafka-tls-px.yaml

```diff
@@ -0,0 +1,47 @@
+---
+apiVersion: kafka.strimzi.io/v1beta1
+kind: Kafka
+metadata:
+  name: px-cluster

```

Comment:
Missing `namespace: kafka`

Reply:
The [script](https://github.com/sayar/fabrikate-kafka-sample/pull/14/files/823415cc0f0f8fbd35ecf20e7efb2084f0b48a28#diff-7387bdeabf9b7c8e9e0b5ad4ee69fd85R37) adds the appropriate namespace. Still, add it here?